DigitalLearningSolutions.Web/ViewModels/RegisterDelegateByCentre/SummaryViewModel.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.RegisterDelegateByCentre
+{
+    using System.Collections.Generic;
+    using System.Globalization;
+    using DigitalLearningSolutions.Web.Models;
+    using DigitalLearningSolutions.Web.ViewModels.Common;
+
+    public class SummaryViewModel
+    {
+        public SummaryViewModel() { }
+
+        public SummaryViewModel(DelegateRegistrationByCentreData data)
+        {
+            FirstName = data.FirstName;
+            LastName = data.LastName;
+            Email = data.Email;
+            Alias = data.Alias;
+            IsPasswordSet = data.IsPasswordSet;
+            PreviousAction = "Password";
+            ShouldSendEmail = data.ShouldSendEmail;
+            if (ShouldSendEmail)
+            {
+                CultureInfo originalCulture = CultureInfo.CurrentCulture;
+                CultureInfo.CurrentCulture = new CultureInfo("en-GB");
+                WelcomeEmailDate = data.WelcomeEmailDate!.Value.ToShortDateString();
+                CultureInfo.CurrentCulture = originalCulture;
+                PreviousAction = "WelcomeEmail";

```

Comment:
This seems a bit hacky :P Can we just specify the date format that we desire (and maybe save it in a date time helper or somewhere as a const so we can reference the same format everywhere)?

Reply:
Yeah I don't know how I missed ToString 🤦‍♂️ .